auger_ml/tasks_queue/celery_app.py

```diff
@@ -0,0 +1,16 @@
+from celery import Celery
+import os
+
+celeryApp = Celery('auger_ml.tasks_queue',
+             include=[
+             'auger_ml.tasks_queue.optimizers_tasks', 
+             'auger_ml.tasks_queue.trials_tasks', 
+             'auger_ml.tasks_queue.AugerMLQueue'])
+
+celeryApp.conf.enable_utc = True
+celeryApp.conf.broker_url = "redis://%s:%s/1" % (
```

Comment:
Why don't we use `/0` database?

```suggestion
celeryApp.conf.broker_url = "redis://%s:%s/0" % (
```

Reply:
yes, make sense